components/function-runtimes/nodejs12/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
```

Comment:
Where it comes from?

Reply:
https://github.com/nodejs/docker-node/blob/main/12/alpine3.12/docker-entrypoint.sh

Reply:
I would add the link as a comment.